nmt.go

```diff
@@ -64,14 +65,16 @@ func IgnoreMaxNamespace(ignore bool) Option {
 
```

Comment:
Github won't let me comment on line 46 because it wasn't changed, but it should be updated from 
```go
```

Reply:
Thanks! Updated in 88ca665